ste/jobStatusManager.go

```diff
@@ -38,12 +38,14 @@ type JobPartCreatedMsg struct {
 
 type xferDoneMsg = common.TransferDetail
 type jobStatusManager struct {
-	js          common.ListJobSummaryResponse
-	respChan    chan common.ListJobSummaryResponse
-	listReq     chan bool
-	partCreated chan JobPartCreatedMsg
-	xferDone    chan xferDoneMsg
-	done        chan struct{}
+	js             common.ListJobSummaryResponse
+	respChan       chan common.ListJobSummaryResponse
+	listReq        chan bool
+	partCreated    chan JobPartCreatedMsg
+	xferDone       chan xferDoneMsg
+	done           chan struct{}
+	processMsg     chan struct{}
+	doneProcessMsg chan struct{}
```

Comment:
For clarity of intent, suggest following re-names for these channels

processMsg         -> drainXferDoneSignal
doneProcessMsg -> xferDoneDrainedSignal

nyc, but I think whenever a channel is used for signalling some event (as opposed to carrying messages) we should use the Signal suffix. Btw, I see that other places are using Channel or Ch suffix, but I think Signal is much better for readability.